### PR TITLE
/efficacy page implementation

### PR DESCRIPTION
**Pull request by adamkecskes**
Asana link: https://app.asana.com/0/1193069161928913/1202001699843255/f

Desktop layout:
effifacy-desktop

Narrow screen layout:
efficacy-narrow

### CODE REVIEW

File: app/views/efficacy/EfficacyView.vue
Comment (nwinter) [inline app/views/efficacy/EfficacyView.vue:28]: Let's put alt tags on all images

File: app/views/efficacy/EfficacyView.vue
Comment (nwinter) [inline app/views/efficacy/EfficacyView.vue:32]: Let's add i18n for all strings

File: app/views/efficacy/EfficacyView.vue
Comment (nwinter) [inline app/views/efficacy/EfficacyView.vue:188]:
```suggestion
    modal-get-licenses(v-if="showModalGetLicenses" @close="showModalGetLicenses = false" subtitle="To get licenses for our research-backed curriculum, send us a message and our classroom success team will be in touch!" email-message="Hi Ozaria! I'm interested in learning more about Ozaria's research-backed computer science curriculum.")
```

File: app/views/efficacy/EfficacyView.vue
Comment (nwinter) [inline app/views/efficacy/EfficacyView.vue:207]: Let's add a font stack here (ideally in a variable) so that it will still work even if we haven't loaded Space Mono. Similar comment to https://github.com/codecombat/codecombat/pull/6605#discussion_r880833410 which was adjusted in https://github.com/codecombat/codecombat/pull/6605/commits/8ce66ea757c191d40426dc1d30a60cc3021f102c#diff-045a09881f4c41285ac1b13c26dc45e489e826d14c040289717ac19bf712337cR5

**Review by nwinter — CHANGES_REQUESTED** [review]
Really great stuff! I didn't test manually yet, just leaving a few quick nits, but overall it looks fantastic, good structure.

All images (that aren't SVGs) should be run through ImageOptim (free download) before being merged in. Sometimes it doesn't help much, but it's easy, and sometimes it significantly reduces filesizes.

Screen Shot 2022-05-26 at 20 13 40

[logo-500px.png](https://github.com/codecombat/codecombat/pull/6637/files#diff-e2d5685d148a102929a4a0e88e7424dc312c2a911154e838ec54a5c8e97012a1) – do we have a usable logo in the repo already, so we don't have to add another? (I also wonder this for crystal image)

[christal.png](https://github.com/codecombat/codecombat/pull/6637/files#diff-8143bd21b699ef85b6c99c3703f8122758e5ddda9cfac7d14c37b4faf1697e8c) – spelled "crystal"

[image-1.png](https://github.com/codecombat/codecombat/pull/6637/files#diff-bbfd089dfa9c7963b5fe259da274bfe34af1548adf47c9f155c5ce486e6aace9) should probably be a little more descriptive, like "classroom-image-1"

File: app/core/vueRouter.ozar.js
Comment (mrfinch) [inline app/core/vueRouter.ozar.js:111]: webpackChunkName should be different 😛

File: app/views/efficacy/EfficacyView.vue
Comment (mrfinch) [inline app/views/efficacy/EfficacyView.vue:1]: why do we need this?

File: app/views/efficacy/EfficacyView.vue
Comment (mrfinch) [inline app/views/efficacy/EfficacyView.vue:913]: generally, I prefer not to put styles on HTML tags directly since if we add a new h3 element in this div, it would pick up the same styles. But this is not blocking the merge of this code.

**Comment by adamkecskes** [comment]
@nwinter I found no similar existing images neither for the logo no for the crystal.
The existing crystal I found is a bit different: https://www.ozaria.com/images/ozaria/home/crystal-art.png
The similar sized Ozaria logo I found was lacking the yellow "sun" from the background.

File: app/views/efficacy/EfficacyView.vue
Comment (adamkecskes) [reply app/views/efficacy/EfficacyView.vue:922]: @mrfinch ok, I'll do so next time.

**Review by nwinter — APPROVED** [review]

**Pull request merged**
